documentation/modules/con-standard-address-space.adoc

```diff
@@ -4,11 +4,11 @@
 
 [id='con-standard-address-space-{context}']
 = Standard Address Space
-The standard address space is the default address space in {ProductName}. It consists of an AMQP router network in combination with attachable storage units. The implementation of a storage unit is hidden from the client and the routers with a well-defined API. This address space type is appropriate when you have many connections and addresses. However, it has the following limitations: 
+The standard address space is the default address space in {ProductName}. It consists of an AMQP router network in combination with attachable storage units. The implementation of a storage unit is hidden from the client and the routers with a well-defined API. This address space type is appropriate when you have many connections and addresses. However, the standard address space has the following limitations:
```

Comment:
I know you didn't add this but 'The implementation of a storage unit is hidden from the client and the routers with a well-defined API.' seems odd to me. What does that mean? @lulf @rgodfrey ?

Reply:
@grs I think it is a bit too abstract :) How about replacing storage units with broker and reword to something like "Clients connect to a message router, which forwards messages to/from one or more message brokers" ?

Reply:
Yes, that is easier to understand.

Reply:
@grs, @lulf, Thanks, I'll edit this to reflect the new verbiage.